### PR TITLE
Tc/display likes

### PR DESCRIPTION
**Pull request by tomcarmichael**
Likes are displayed within each post. 
If the user has not yet liked the post they see the correct colour thumb emoji and clicking it adds their username to the likes array in the DB

### CODE REVIEW

File: frontend/src/components/Likes/Likes.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useState } from 'react';
+import likedThumbsUp from '../../images/likedThumbsUp.png';
+import unLikedThumbsUp from '../../images/unlikedThumbsUp.png';
+
+const Likes = ({ likes, parent }) => {
+
+  // const [alreadyLiked, setAlreadyLiked] = useState(false)
```
Comment (pablisch) [inline frontend/src/components/Likes/Likes.js:7]: Could this be deleted?

File: frontend/src/components/feed/Feed.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ const Feed = ({ navigate }) => {
           window.localStorage.setItem("token", data.token)
           setToken(window.localStorage.getItem("token"))
           data.posts.forEach((post) => {
+            console.log(post)
```
Comment (pablisch) [inline frontend/src/components/feed/Feed.js:27]: Is this still needed?

**Review by pablisch — APPROVED** [review]
As tasty as a last-day retro! 
I put a couple of comments about lines that might not be needed but nothing to stop this from going through. Looking forward to seeing it pulled!

File: frontend/src/components/Likes/Likes.js (hunk 1 above)
Comment (tomcarmichael) [reply frontend/src/components/Likes/Likes.js:7]: Definitely .

File: frontend/src/components/feed/Feed.js (hunk 2 above)
Comment (tomcarmichael) [reply frontend/src/components/feed/Feed.js:27]: no siree :) - good shout

**Pull request merged**